### ISSUE TITLE
Autohoming with 3D Touch Z-axis doesn't go down

### ISSUE THREAD

**Issue by iRaiFx** [mechano/Kingroon_KP5L_Marlin_Firmware#7]
I had a problem with Z-axis which doesn't go down when homing

I changed the line 

#ifdef USE_BLTOUCH
  #define Z_MIN_PROBE_PIN Z_MAX_PIN
#endif

to

#ifdef USE_BLTOUCH
  #define Z_MIN_PROBE_PIN Z_MIN_PIN
#endif

**Comment by mechano** [comment]
Hi there. 
The  # symbol in front of the line means it's commented. 

So what Configuration.h did you modify the MBL or ABL version?

**Comment by iRaiFx** [comment]
ABL_VLCD version.

I tried flashing the same bin file multiple times, everything works except for when the Z axis homes
Which probes in the air strangely, so i got a new 3D Touch probe and still same issue.

Then i took the provided Configuration.h and changed that exact line and strangely it worked.
I followed the tutorial for plug in the 3D Touch in the previous Z-endstop plug (Z-min)

Now it works greatly, thank you Mr.Mechano
